hack/postrelease/test_manifests_archives.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import tarfile
+from nose.tools import with_setup
+
+from versions import RELEASE_VERSION
+
+url = "https://github.com/projectcalico/calico/releases/download/{v}/release-{v}.tgz".format(
+    v=RELEASE_VERSION
+)
+
+manifest_list = [
+    "calico.yaml",
+    "calico-etcd.yaml",
+    "calico-bpf.yaml",
+    "calico-typha.yaml",
+    "calico-vxlan.yaml",
+    "calico-windows-bgp.yaml",
+    "calico-windows-vxlan.yaml",
+    "calicoctl.yaml",
+    "calicoctl-etcd.yaml",
+    "canal.yaml",
+    "canal-etcd.yaml",
+    "tigera-operator.yaml",
+    "custom-resources.yaml",
+]
+
+
+def setup_archive():
+    response = requests.get(url, stream=True)
+    global file
+    file = tarfile.open(fileobj=response.raw, mode="r|gz")
+
+
+def teardown_archive():
+    file.close()
+
+
+@with_setup(setup=setup_archive, teardown=teardown_archive)
+def test_manifest_present():
+    for manifest in manifest_list:
+        yield check_manifest_present, manifest
```

Comment:
I don't think a `yield` is the right execution method here - there's never a situation where we're wanting to pause execution of check_manifest_present and then start where we left off, right?

Probably better to just use a function call - `check_manifest_present(manifest)`

Reply:
i'm using nosetest[ test generators](https://nose.readthedocs.io/en/latest/writing_tests.html#test-generators) which creates individual tests for each manifest - more explict with failure

a function call gives this result summary

```sh
test_manifests_archives.test_manifest_present ... ok (23.9096s)
```

comapred to a yieded tests

```sh
test_manifests_archives.test_manifest_present('calico.yaml',) ... ok (23.4394s)
test_manifests_archives.test_manifest_present('calico-etcd.yaml',) ... ok (0.0001s)
test_manifests_archives.test_manifest_present('calico-bpf.yaml',) ... ok (0.0003s)
test_manifests_archives.test_manifest_present('calico-typha.yaml',) ... ok (0.0002s)
test_manifests_archives.test_manifest_present('calico-vxlan.yaml',) ... ok (0.0002s)
test_manifests_archives.test_manifest_present('calico-windows-bgp.yaml',) ... ok (0.0003s)
test_manifests_archives.test_manifest_present('calico-windows-vxlan.yaml',) ... ok (0.0002s)
test_manifests_archives.test_manifest_present('calicoctl.yaml',) ... ok (0.0002s)
test_manifests_archives.test_manifest_present('calicoctl-etcd.yaml',) ... ok (0.0002s)
test_manifests_archives.test_manifest_present('canal.yaml',) ... ok (0.0002s)
test_manifests_archives.test_manifest_present('canal-etcd.yaml',) ... ok (0.0002s)
test_manifests_archives.test_manifest_present('tigera-operator.yaml',) ... ok (0.0002s)
test_manifests_archives.test_manifest_present('custom-resources.yaml',) ... ok (0.0002s)
```

Reply:
Ahhh, gotcha. Unfortunate that we still have Python tests hanging around but maybe one day we'll switch these all over to go. Good for now.